crates/holochain/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Refactor Mr. Bundle which affects the manifest types in `holochain_types`.
+- Existing app bundles cannot be used. You can either rebundle existing happs with `hc-bundle` and the `--raw` flag or 
```

Comment:
```suggestion
- **BREAKING** Existing app bundles cannot be used. You can either rebundle existing happs with `hc-bundle` and the `--raw` flag or 
```
Not sure if this would really be considered breaking, but seems important to call out more prominently.